source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +224,53 @@ export default class WindowManager {
       return
     }
 
+    const platformIcons: {[prop: string] : string} = {
+      'darwin': '/png/22x22_white.png',
+      'win32': '/icon.ico'
+    }
+
+    this._mainWindow.on('show', async () => {
+      if (this._tray == null) {
+        if (process.platform === 'linux') {
+          let size = '16'
```

Comment:
16 is a very small default. Most platforms have a task bar around 32. I suggest make 32.
```typescript
          let size = '32'
```